gpAux/gpdemo/README

```diff
@@ -46,7 +46,7 @@ RUNNING GP DEMO
 
 4. Create the cluster:
 
-	make cluster
+	./gpdemo.sh
```

Comment:
Using Makefile should be ok during develop,
-> './gpdemo.sh or make clsuter' ? is better?